internal/server/admin/certificate_service.go

```diff
@@ -35,6 +82,11 @@ func (s *CertificateService) GetCertificate(ctx context.Context,
 	if err != nil {
 		return nil, s.err(ctx, err)
 	}
+	if !util.IsReference(result.Certificate.Cert) {
+		if err := extractCertInfo(result.Certificate); err != nil {
+			return nil, s.err(ctx, err)
```

Comment:
No need to use `s.err()` when calling `extractCertInfo()` as it won't be outputting any public-facing error anyway. Same goes for the `certificatesFromObjects()` call too.

Reply:
Can we ensure that's fixed elsewhere too? Obviously, this comment will be moot once [this](https://github.com/Kong/koko/pull/545#discussion_r1046395371) is done though.